injector/network_disruption.go

```diff
@@ -769,7 +769,7 @@ func (i *networkDisruptionInjector) watchServiceChanges(watcher serviceWatcher,
 			if !ok { // channel is closed
 				watcher.kubernetesServiceWatcher = nil
 			} else {
-				i.config.Log.Infow(fmt.Sprintf("changes in service %s/%s", watcher.watchedServiceSpec.Name, watcher.watchedServiceSpec.Namespace), "eventType", event.Type)
+				i.config.Log.Debugw(fmt.Sprintf("changes in service %s/%s", watcher.watchedServiceSpec.Name, watcher.watchedServiceSpec.Namespace), "eventType", event.Type)
```

Comment:
This happen extremely often at the beginning of a disruption, and as long as they're working (DEBUG), we don't need to see them.